src/Symfony/Component/Ldap/Adapter/ExtLdap/Query.php

```diff
@@ -174,8 +174,7 @@ public function getResources(): array
     private function resetPagination()
     {
         $con = $this->connection->getResource();
-        $this->controlPagedResultResponse($con, 0, '');
-        $this->serverctrls = [];
```

Comment:
the method `controlPagedResultResponse` does not reset this variable for php < 7.3